### PR TITLE
NathanBeals.WinSSH-Pageant version 2.1.0

### PR DESCRIPTION
**Pull request by ndbeals**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55240)

### CODE REVIEW

File: manifests/n/NathanBeals/WinSSH-Pageant/2.1.0/NathanBeals.WinSSH-Pageant.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- interactive
+- silent
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerLocale: en-US
+  Scope: user
+  InstallerUrl: https://github.com/ndbeals/winssh-pageant/releases/download/v2.1/winssh-pageant-v2.1_amd64.msi
+  InstallerSha256: 95DC1159BB9F58EF2DB7E3D201C6B23E093192154C50E5356440B7F35D432661
+  ProductCode: '{839A2310-DE92-47F9-B5DB-8F1E41ABA405}'
```
Comment (OfficialEsco) [inline manifests/n/NathanBeals/WinSSH-Pageant/2.1.0/NathanBeals.WinSSH-Pageant.installer.yaml:20]:
```suggestion
  ProductCode: '{F7CDE09D-86C1-4AE9-A3E9-6F374DF491E2}'
```

File: manifests/n/NathanBeals/WinSSH-Pageant/2.1.0/NathanBeals.WinSSH-Pageant.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Architecture: x86
+  InstallerLocale: en-US
+  Scope: user
+  InstallerUrl: https://github.com/ndbeals/winssh-pageant/releases/download/v2.1/winssh-pageant-v2.1_386.msi
+  InstallerSha256: 8FC8121E688D0208BBE6CBD44C03AB9F1D617F21FF74FA746D716EDBCC414F43
+  ProductCode: '{B5A49790-199C-4BA6-8E4A-B28CA56A04F8}'
```
Comment (OfficialEsco) [inline manifests/n/NathanBeals/WinSSH-Pageant/2.1.0/NathanBeals.WinSSH-Pageant.installer.yaml:26]:
```suggestion
  ProductCode: '{BE8EAE9B-131A-4152-B667-72FCEC203015}'
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — DISMISSED** [review]

**Comment by ndbeals** [comment]
Thanks @OfficialEsco, That ProductCode thing was very confusing to me last night. I think something went wrong with the verification pipeline? it quit running after 1 second?

**Review by OfficialEsco — APPROVED** [review]
Yeap thats another clever way to do it =)

**Pull request merged**